src/modules/31-filestore/pages/filestore/__tests__/FileStorePage.test.tsx

```diff
@@ -29,6 +29,7 @@ jest.mock('services/cd-ng', () => ({
       mutate: jest.fn().mockImplementation(() => Promise.resolve(rootMock))
     }
   }),
+  useGetSettingValue: jest.fn().mockImplementation(() => ({ data: { value: 'true' } })),
```

Comment:
Not an issues, but you can simplify these things a bit with `mockReturnValue`
```suggestion
  useGetSettingValue: jest.fn().mockReturnValue({ data: { value: 'true' } }),
```